Backend/Underdogs-Struggling/src/test/resources/application-test.properties

```diff
@@ -0,0 +1,7 @@
+spring.datasource.url=jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;MODE=MySQL
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=root
+spring.datasource.password=
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+spring.jpa.show-sql=true
+spring.jpa.hibernate.ddl-auto=create-drop
```

Comment:
Why use H2 DB as test database? We are using MySQL as the primary dayabase.

Reply:
It is a in-memory database for isolated unit test, which avoids the recovery of our Mysql database, and is convenient for future deployment testing ignoring the concurrency conflicts.

Reply:
Currently, the biz level unit test and validation part is still WIP, but we need an in-memory database to isolate our production database environment for sure.

Reply:
One more inconsistency here. 

Not only unit test, the test, dev and prd environment are not set yet. The only thing we have for now is the local database running on our own machine. Whether it is local testing or building other environments in the future, we should choose the same database. This is not related to the production environment isolation.